test/unit/attendance.ts

```diff
@@ -0,0 +1,22 @@
+import { markAttendance } from '../../src/components/attendance/service';
+jest.mock('../../src/components/attendance/repository')
+import { saveAttendance } from '../../src/components/attendance/repository';
+
+describe('Mark attendance', () => {
+  it('should throw error if user is inactive', async () => {
+    const payload = {
+      userId:  123456,
+      checkIn: '02:00'
+    }
+
+    await expect(markAttendance(payload)).rejects.toThrowError('user inactive');
```

Comment:
better to break it down in 3 steps:
1. Given or Arrange
2. When or Act
3. Then or Assert

call mark attendance outside of expect and then pass the result in expect.